### PR TITLE
refactor(@ui/settings): add profile settings to storybook

### PR DESCRIPTION
**Pull request by iurimatias**
### What does the PR do

initial add of the profile setttings. needs second iteration afterwards, in particular for the MyProfilePreview component that is currently a bit too coupled

### CODE REVIEW

File: storybook/pages/MyProfileViewPage.qml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+                function uploadImage(source, aX, aY, bX, bY) {
+                    logs.logEvent("profileStore::uploadImage", ["source", "aX", "aY", "bX", "bY"], arguments)
+                }
+
+                function removeImage() {
+                    logs.logEvent("profileStore::removeImage")
+                }
+
+                function getQrCodeSource(publicKey) {
+                    logs.logEvent("profileStore::getQrCodeSource", ["publicKey"], arguments)
```
Comment (caybro) [inline storybook/pages/MyProfileViewPage.qml:198]: This should return something

File: storybook/pages/MyProfileViewPage.qml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                icon: "userProfile.icon"
+                userDeclinedBackupBanner: true
+
+                onUserDeclinedBackupBannerChanged: {
+                    logs.logEvent("profileStore::signals::onUserDeclinedBackupBannerChanged")
+                }
+
+                // TODO: replace later with contact details
+                // details: Utils.getContactDetailsAsJson(pubkey)
+                details: "{}"
```
Comment (caybro) [inline storybook/pages/MyProfileViewPage.qml:187]: Look at the mock in the ProfileDialogViewPage

File: storybook/pages/MyProfilePreviewPage.qml [hunk 3]
```diff
@@ -0,0 +1,189 @@
+import QtQuick 2.14
```
Comment (caybro) [inline storybook/pages/MyProfilePreviewPage.qml:1]: There is already the ProfileDialogViewPage

File: storybook/pages/MyProfilePreviewPage.qml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+        MyProfilePreview {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            contactsStore: ContactsStore {
+                myPublicKey: "0x123"
+
+                function hasReceivedVerificationRequestFrom(publicKey) {
+                    logs.logEvent("contactsStore::hasReceivedVerificationRequestFrom", ["publicKey"], arguments)
```
Comment (caybro) [inline storybook/pages/MyProfilePreviewPage.qml:27]: This should return something (`true/false`)

**Review by caybro — CHANGES_REQUESTED** [review]
Some minor things

File: storybook/pages/MyProfilePreviewPage.qml [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        MyProfilePreview {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            contactsStore: ContactsStore {
```
Comment (jrainville) [inline storybook/pages/MyProfilePreviewPage.qml:23]: As Michal pointed out in my PR here https://github.com/status-im/status-desktop/pull/8010#pullrequestreview-1152372653 , you should probably use a stub instead. It really made it simpler.

**Review by jrainville — COMMENTED** [review]
I think using stub stores will simplify your life and reduce the risk of having errors or warnings pop up

**Pull request closed without merging**